### PR TITLE
fixed builder issue

### PR DESCRIPTION
**Pull request by NazarShchur**
fixed builder

### CODE REVIEW

File: src/main/java/com/robosh/ejournal/entity/Parent.java [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Entity;
 import javax.persistence.ManyToMany;
 import java.util.List;
 
+@EqualsAndHashCode(callSuper = true)
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/Parent.java:13]: Is it needed here, and why?

File: src/main/java/com/robosh/ejournal/entity/School.java [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import lombok.Builder;
-import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.OneToOne;
 
-@Data
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/School.java:12]: Why did you remove Data? It contains needed functional like equals and hashcode

File: src/main/java/com/robosh/ejournal/entity/Teacher.java [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
 @Entity
 public class Teacher extends User {
+
     @OneToOne
     private Group group;
 
     private String description;
 
-    @Builder
+    @Builder(builderMethodName = "")
```
Comment (Ostape) [inline src/main/java/com/robosh/ejournal/entity/Teacher.java]: why you put empty string here

Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/Teacher.java]: Is it needed to set builderMethodName? Only Builder will be enough

**Review by SashaRoyenko — CHANGES_REQUESTED** [review]
Change the issues. I think we need Data annotation

File: src/main/java/com/robosh/ejournal/entity/Parent.java (hunk 1 above)
Comment (NazarShchur) [reply src/main/java/com/robosh/ejournal/entity/Parent.java:13]: it is needed because IDE told me
